hw1/02-palindrome.js

```diff
@@ -1,3 +1,41 @@
-const elem = document.querySelector('input');
+// Declarations
+const elem = document.querySelector("input");
+elem.addEventListener("change", handleInput); // get input on enter
+const output = document.getElementById("result"); // output the result to HTML
 
-elem.addEventListener('input', handleInput);
+// Check for positive number
+function isPositiveNumber(num) {
+  if (num < 0) {
+    return false;
+  }
+}
+
+// Reverse a number
+function reverseNumber(num) {
+  let result = "";
+  for (i = String(num).length - 1; i >= 0; --i) {
+    result = result + num[i];
+  }
+  return result;
+}
+
+function handleInput() {
+  // Element to string
+  const userInput = elem.value;
+
+  // Check for a positive number
+  if (isPositiveNumber(userInput) === false) {
+    window.alert("Please, enter a positive number.");
+    return;
+  }
+
+  // Get the reverse of the userInput
+  const reverse = reverseNumber(userInput);
+
+  // Compare the reverse with original for result
+  if (userInput === reverse) {
+    output.innerHTML = "Yes. " + userInput + " is a palindrome.";
```

Comment:
The Airbnb documentation mentions in **Rule 6.3** that you should use template strings instead of concatenation. 
**Ex:**
// bad
function sayHi(name) {
  return 'How are you, ' + name + '?';
}
// good
function sayHi(name) {
  return `How are you, ${name}?`;
}

Reply:
Don't forget to add text color to your outputs. Red for failures, green for success. Using the textContent command should replace the need for the innerHTML command.

Reply:
Got it! done!